tests/compiletest.rs

```diff
@@ -13,11 +13,16 @@ macro_rules! eprintln {
     }
 }
 
+#[cfg(debug_assertions)]
+const MIRI_PATH: &str = "target/debug/miri";
+#[cfg(not(debug_assertions))]
+const MIRI_PATH: &str = "target/release/miri";
```

Comment:
I pushed a commit that hopefully should fix this. (#rust IRC channel was very helpful :) )